src/creature.h

```diff
@@ -185,6 +185,11 @@ class Creature
         dealt_projectile_attack projectile_attack( const projectile &proj, const tripoint &target,
                                                    double total_dispersion );
 
+        /**
+         * Return the chance that a projectile attack will hit with at least the given accuracy
```

Comment:
`[0..1]` or `(0..1)`. Can you document this here as it could be interpreted as reutrning a percentage chance?